src/js/components/Button/doc.js

```diff
@@ -44,6 +45,22 @@ export const doc = Button => {
     as: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).description(
       `The DOM tag or react component to use for the element.`,
     ),
+    badge: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.element,
+      PropTypes.number,
+      PropTypes.shape({
+        background: backgroundPropType,
+        max: PropTypes.number,
+        value: PropTypes.number,
+      }),
+    ])
+      .description(
+        `An indicator to show on the top-right of the Button. For accessibility,
+         add an aria-label that provides a useful message to screen readers. 
+         For example, "2 unread notifications".`,
```

Comment:
Are we encouraging callers to use `aria-label` over `a11yTitle`? I know we are supporting `aria-label` but `a11yTitle` is the documented prop.

Also, would this be more clear to reference "add an a11yTitle to the Button...?" If I wasn't as familiar, I might assume that was something I should call from the badge.
```suggestion
        `An indicator to show on the top-right of the Button. For accessibility,
         supplement the badge by adding an a11yTitle to the Button which 
         provides a useful message to screen readers. For example, "2 unread 
         notifications".`,
```

Reply:
Done. Yes, a11yTitle is the one we document so we should use that.

Reply:
Done.